### DISCUSSION TITLE
Sulu form, trying to add default value for block field

### DISCUSSION THREAD

**Discussion by niji-rmejean** [sulu/sulu#6983] [Q&A] [answered]
Hi,
I'm trying to add a default value for my block field in a custom entity, but I did not find any mention of a way to do that in the sulu documentation. Is there a way to do that, maybe a hook or something like that ?

Thanks by advance !

**Comment by alexander-schranz** [comment, marked as answer]
@niji-rmejean currently there is no way to define a default value beside for single_select.

Still you can create a document managerlistener on persist do manipulate data or even use symfony request listener to manipulate such things.
